server/services/__tests__/bodyProfile.test.ts

```
@@ -0,0 +1,237 @@
+  describe('deleteBodyProfile', () => {
+    it('should delete body profile successfully', async () => {
+      const auth0Id = 'auth0|123'
+      const userId = 'user-id'
+
+      const mockUser = { id: userId }
+      const existingProfile = {
+        id: 'profile-id',
+        userId,
+      }
+
+      jest.spyOn(bodyProfileService['userRepository'], 'findByAuth0Id').mockResolvedValue(mockUser as any)
```

Biome (1.9.4)

[error] The computed expression can be simplified without the use of a string literal.

Unsafe fix: Use a literal key instead.


(lint/complexity/useLiteralKeys)